library-best-practices/glossary.md

```diff
@@ -11,7 +14,14 @@ Java Dependency Glossary
   `LinkageError`, or other related exceptions.
   - Sub-type: **Linkage conflict**
   - Sub-type: **Missing class error**
-
+  - Sub-type: **Static linkage error**: A linkage error caused by a direct code
+    reference (non-reflective).
+  - Sub-type: **Dynamic linkage error**: A linkage error caused by a reflective
```

Comment:
maybe "a reflective reference" --> reflection

Reply:
This is just mirroring the definition of "dynamic linkage conflict" below - I'd prefer to fix up this wording separately.